Doc/whatsnew/3.8.rst

```diff
@@ -788,6 +788,9 @@ The following features and APIs have been removed from Python 3.8:
 * The function :func:`platform.popen` has been removed, it was deprecated since
   Python 3.3: use :func:`os.popen` instead.
 
+* The function :func:`time.clock` has been removed, it was deprecated since Python
+  3.3: use :func:`time.perf_counter` or :func:`time.process_time` instead.
```

Comment:
Hum, you should copy-paste the end of the sentence :-) Add: ", depending on your requirements, to have a well defined behaviour".

Reply:
Oh. I'm the original author of "behaviour" spelling. I have been told in the meanwhile that "behaviour" is the UK spelling, but Python doc prefers US spelling: "behavior" ;-)

Reply:
Done. Thanks.